pyproject.toml

```diff
@@ -181,6 +181,7 @@ requires = ["poetry-core>=1.2.1"]
 ignore_missing_imports = true
 python_version = "3.9"
 plugins = ["pydantic.mypy"]
+explicit_package_bases = true
```

Comment:
This ensures that only these [dirs](https://github.com/basetenlabs/truss/blob/e533c3f48586231264565d65b973404ed157a4b2/pyproject.toml?plain=1#L18-L22) are included in the built wheel?